web/src/main/java/com/synopsys/integration/alert/web/api/config/ConfigActions.java

```diff
@@ -186,7 +187,9 @@ public ConfigActions(AuthorizationManager authorizationManager, DescriptorAccess
 
     @Override
     protected ValidationActionResponse validateWithoutChecks(FieldModel resource) {
-        List<AlertFieldStatus> fieldStatuses = fieldModelProcessor.validateFieldModel(resource);
+        Optional<GlobalValidator> globalValidator = fieldModelProcessor.getGlobalValidator(resource);
+        List<AlertFieldStatus> fieldStatuses = (globalValidator.isPresent()) ? globalValidator.get().validateFieldModel(resource) : fieldModelProcessor.validateFieldModel(resource);
```

Comment:
Should this be done with an `ifPresentOrElse`?

Reply:
If I were to use that method I'd need to create a final list that I then add items to in scope. This way seemed the most straight forward